### PR TITLE
Feature/fix query for relationships

### PR DESCRIPTION
**Pull request by hendrikbeneke**

### CODE REVIEW

File: lib/jsonApiTranslator.js [hunk 1]
```diff
@@ -115,6 +115,13 @@ function transformQuery(req) {
         }
         return value
     }
+    function transformRelationships(key) {
+        const relationships = schema.relationships || {};
+        if(relationships[key]){
+            return `relationships.${key}.data.id`;
+        }
+        return key;
```
Comment (ssebro) [inline lib/jsonApiTranslator.js:123]: seems like the ret types might mismatch here/be surprizing here, but this may be a misread.

Comment (ssebro) [reply lib/jsonApiTranslator.js:123]: ok, your slack comment clears this up.

**Review by ssebro — COMMENTED** [review]
lgtm

**Review by blabno — APPROVED** [review]
Integration tests would be nice here

**Pull request merged**